compilers/concrete-compiler/compiler/lib/Support/TFHECircuitKeys.cpp

```diff
@@ -105,6 +105,12 @@ TFHECircuitKeys extractCircuitKeys(mlir::ModuleOp moduleOp) {
     secretKeys.insert(op.getKeyAttr().getOutputKey());
   });
 
+  moduleOp->walk([&](TFHE::BatchedKeySwitchGLWEOp op) {
+    keyswitchKeys.insert(op.getKeyAttr());
+    secretKeys.insert(op.getKeyAttr().getInputKey());
+    secretKeys.insert(op.getKeyAttr().getOutputKey());
+  });
```

Comment:
Same here, I just fixed the minimum to make the rewritten Parametrization pass work. But I agree that this should be fixed, probably in a followup commit. It may even be worth to have a `BootstrapLike` op interface to unify these kinds of things.

Reply:
Tracked in Issue https://github.com/zama-ai/concrete/issues/685.